Dockerfile.ocp

```diff
@@ -2,18 +2,15 @@ FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-base-openshift AS builder
 
 WORKDIR /tmp
 
-## NOTE(derekh): We need to build our own grub image because the one
-## that gets installed by grub2-efi-x64 (/boot/efi/EFI/centos/grubx64.efi)
-## looks for grub.cnf in /EFI/centos, ironic puts it in /boot/grub
 RUN if [ $(uname -m) = "x86_64" ]; then \
-      dnf install -y gcc git make genisoimage xz-devel grub2 grub2-efi-x64-modules shim dosfstools mtools && \
+      dnf install -y gcc git make genisoimage xz-devel grub2 grub2-efi-x64 shim dosfstools mtools && \
```

Comment:
you can probably remove the packages: gcc git make genisoimage xz-devel

Reply:
Yeah, I was considering a follow-up, did not want to blend in unrelated changes though